packages/cozy-client/src/store/realtime.js

```diff
@@ -54,9 +47,23 @@ const dispatchChange = (
  * @param {object} client - CozyClient instance
  * @param {import("../types").Doctype} doctype - Doctype of the document to create
  * @param {import("../types").CouchDBDocument} couchDBDoc - Document to create
+ * @param {object} [options] Options
+ * @param {function} [options.computeCreatedDoc] Optional function to transform the normalized document before dispatch
```

Comment:
`computeCreatedDoc` sounds ok, but maybe a `applyAttributeComputation` would be clearer: it better suggests that it is a function, you can reuse it whether it is a create/update/delete, and it explicit mention it is used for attribute computation.

Reply:
Good point 👍 I used the term enhance to mean the improvement of an existing object. @paultranvan does it seems more clearer ?